samples/subsys/llext/modules/sample.yaml

```diff
@@ -0,0 +1,34 @@
+common:
+  tags: llext
+  arch_allow:
+    - arm
+    - xtensa
+  filter: not CONFIG_MPU and not CONFIG_MMU and not CONFIG_SOC_SERIES_S32ZE
+  platform_exclude:
+    - apollo4p_evb
+    - apollo4p_blue_kxr_evb
+    - numaker_pfm_m487 # See #63167
+  harness: console
+sample:
+  name: CONFIG_MODULES test
+  description: Call code directly and from extensions
+tests:
+  sample.llext.modules_enabled:
+    extra_configs:
+      - CONFIG_MODULES=y
```

Comment:
just to make sure this is correct - it's also set in prj.conf and left unchanged below in the monolithic case - is that correct?

Reply:
Thanks, you are right, it's an useless line. The point is to have `MODULES` always enabled during this test so I will remove this.